### DISCUSSION TITLE
VSpeak_ECU Widget

### DISCUSSION THREAD

**Discussion by kochpet** [EdgeTX/edgetx#3144] [LUA]
Hi all,
I have a problem with a widget that was originally written for OpenTX and I hope someone can help me. The widget I'm talking about reads turbine telemetry values ​​and displays them graphically. Now that I have switched from OpenTX to EdgeTX, I would like to continue using the widget, but since I cannot program LUA, I hope to find someone here who can take care of the problem. 

Thanks in advance,
Peter
[VSpeak_ECU.zip](https://github.com/EdgeTX/edgetx/files/10581943/VSpeak_ECU.zip)

**Comment by pfeerick** [comment]
I've modified the widget to not mess with the system colors like it was. Judging from the readme in https://www.vspeak-modell.de/en/downloads it looks like it was either meant to be shown full screen (with the top bar still visible), or in a reduced status messages only form? If so, probably just need to do a little tweaking now with size and positioning of stuff. Assuming the widget otherwise works properly?

[vspeak_ecu_main.zip](https://github.com/EdgeTX/edgetx/files/10608301/vspeak_ecu_main.zip)

**Reply by kochpet** [reply]
Thanks very much Peter, you are my hero!
